tests/acceptance/apps/sidebar-test.js

```diff
@@ -56,7 +56,7 @@ test('includes breadcrumb to parent stack', function(assert) {
   signInAndVisit(`/apps/${appId}`);
 
   andThen(function() {
-    let stackHandle = find(`h3:contains(${stackHandle})`);
+    let stackHandle = find(`h3`); // TODO this should read :contains(somehandle)
```

Comment:
This should be ok after a merge. It was addressed in #772. In addition to the JSHint error the assertion below is a false-positive (it is essentially: `assert.ok([], 'has stack link')`).